pkg/email/templates/issue_lcid.gohtml

```diff
@@ -0,0 +1,6 @@
+<p>Lifecycle ID: {{.LifecycleID}}</p>
+<p>Expiration Date: {{.ExpiresAt}}</p>
+<p>Scope: {{.Scope}}</p>
+<p>Next Steps: {{.NextSteps}}</p>
```

Comment:
Next steps is an optional field. Is there a conditional we can add here to only write out the first part of the paragraph if the next steps are included? It's also totally ok to push that to a separate PR if it's going to take some research. This work is still an improvement!